content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -0,0 +1,164 @@
+2. Export the shell variable _MDM_IP_ in a comma-separated list using `export MDM_IP=xx.xxx.xx.xx,xx.xxx.xx.xx`, where xxx represents the actual IP address in your environment. This list contains the IP addresses of the MDMs.
+3. Install the SDC per the _Dell PowerFlex Deployment Guide_:
+    - For Red Hat Enterprise Linux and CentOS, run `rpm -iv ./EMC-ScaleIO-sdc-*.x86_64.rpm`, where * is the SDC name corresponding to the PowerFlex installation version.
+4. To add more MDM_IP for multi-array support, run `/opt/emc/scaleio/sdc/bin/drv_cfg --add_mdm --ip 10.xx.xx.xx.xx,10.xx.xx.xx`1. Create namespace.
+   Execute `kubectl create namespace test-vxflexos` to create the `test-vxflexos` namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'test-vxflexos'.
+
+#### Create Secret
+1. Create namespace: 
+   Execute `kubectl create namespace test-vxflexos` to create the test-isilon namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'test-vxflexos'.
+2. Prepare the config.yaml for driver configuration.
+
+    Example: config.yaml
+
+     ```yaml
+      # Username for accessing PowerFlex system.	
+      # Required: true
+     - username: "admin"
+      # Password for accessing PowerFlex system.	
+      # Required: true
+      password: "password"
+      # System name/ID of PowerFlex system.	
+      # Required: true
+      systemID: "ID1"
+      # REST API gateway HTTPS endpoint/PowerFlex Manager public IP for PowerFlex system.
+      # Required: true
+      endpoint: "https://127.0.0.1"
+      # Determines if the driver is going to validate certs while connecting to PowerFlex REST API interface.
+      # Allowed values: true or false
+      # Required: true
+      # Default value: true
+      skipCertificateValidation: true 
+      # indicates if this array is the default array
+      # needed for backwards compatibility
+      # only one array is allowed to have this set to true 
+      # Required: false
+      # Default value: false
+      isDefault: true
+      # defines the MDM(s) that SDC should register with on start.
+      # Allowed values:  a list of IP addresses or hostnames separated by comma.
+      # Required: true
+      # Default value: none 
+      mdm: "10.0.0.1,10.0.0.2"
+      # Defines all system names used to create powerflex volumes
+      # Required: false
+      # Default value: none
+      AllSystemNames: "name1,name2"
+    - username: "admin"
+      password: "Password123"
+      systemID: "ID2"
+      endpoint: "https://127.0.0.2"
+      skipCertificateValidation: true 
+      mdm: "10.0.0.3,10.0.0.4"
+      AllSystemNames: "name1,name2"
+    ```
+
+    After editing the file, run this command to create a secret called `vxflexos-config`
+    `kubectl create secret generic vxflexos-config -n <driver-namespace> --from-file=config=config.yaml`
```

Comment:
It would make sense to mention namespace test-vxflexos, because we already mentioned it couple of times at the start.

Reply:
addressed